### PR TITLE
Make timeout flag dependent on operating system

### PR DESCRIPTION
**Pull request by rakonn**
## Introduced Changes

Timeout flag of ping is dependent on the os. Under MacOs ping doesn't have the -w flag, but has an equivalent -t flag. Changed this flag depending on the operating system.

Fixes #

## How to Test

`./pepsi ping` on a linux and macos machine.

### CODE REVIEW

File: crates/nao/src/lib.rs [hunk 1]
```diff
@@ -34,10 +34,15 @@ impl Nao {
         host: Ipv4Addr,
         timeout_seconds: u32,
     ) -> Result<Self> {
+        #[cfg(target_os = "macos")]
+        let timeout_flag = "-t";
+        #[cfg(not(target_os = "macos"))]
+        let timeout_flag = "-w";
```
Comment (h3ndrk) [inline crates/nao/src/lib.rs:40]: These could be constants, right?

```suggestion
        #[cfg(target_os = "macos")]
        const TIMEOUT_FLAG: &str = "-t";
        #[cfg(not(target_os = "macos"))]
        const TIMEOUT_FLAG: &str = "-w";
```

**Review by h3ndrk — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! 🎉

I have only one small question and remark. Feel free to implement or discuss below 😉

File: crates/nao/src/lib.rs (hunk 1 above)
Comment (rakonn) [reply crates/nao/src/lib.rs]: Yes they could be. I made the change in the latest commit to this branch.

**Review by h3ndrk — APPROVED** [review]
Thank you very much! 🎉

**Pull request merged**